### PR TITLE
update publisher.md with GetConfiguredNodesOnEndpoint dm doc

### PR DESCRIPTION
**Pull request by dacolgit**

### CODE REVIEW

File: docs/modules/publisher.md [hunk 1]
```diff
@@ -282,19 +282,22 @@ UnpublishNodes method enables a client to remove nodes from a previously configu
...
 
-  *Response:*
+  *Response:* list of OpcNodes configured for the selected Endpoint (and optional parameters).
 
-  *Exceptions:*
+  *Exceptions:* an exception is thrown when method call returned a status othen than 200.
 
   *Example:*
+  >*Method Name:* `GetConfiguredNodesOnEndpoints_V1`\
+  *Payload:* `{"EndpointUrl": "opc.tcp://192.168.100.20:50000"}`
```
Comment (cristipogacean) [inline docs/modules/publisher.md]: can you add an example of response as well?

Comment (dacolgit) [reply docs/modules/publisher.md]: added

File: docs/modules/publisher.md [hunk 2]
```diff
@@ -282,19 +282,35 @@ UnpublishNodes method enables a client to remove nodes from a previously configu
...
+  *Response:* list of OpcNodes configured for the selected Endpoint (and optional parameters).
 
-  *Exceptions:*
+  *Exceptions:* an exception is thrown when method call returned a status othen than 200.
 
   *Example:*
+  >*Method Name:* `GetConfiguredNodesOnEndpoints_V1`\
+  *Payload:* `{"EndpointUrl": "opc.tcp://192.168.100.20:50000"}`\
+  *Response:*  
+  >```
```
Comment (karok2m) [inline docs/modules/publisher.md:305]: Please specify `json` as the language for the code block. i.e: `` ``` `` -> `` ```json ``

File: docs/modules/publisher.md [hunk 3]
```diff
@@ -282,19 +282,35 @@ UnpublishNodes method enables a client to remove nodes from a previously configu
...
 
-  *Response:*
+  *Response:* list of OpcNodes configured for the selected Endpoint (and optional parameters).
 
-  *Exceptions:*
+  *Exceptions:* an exception is thrown when method call returned a status othen than 200.
 
   *Example:*
+  >*Method Name:* `GetConfiguredNodesOnEndpoints_V1`\
+  *Payload:* `{"EndpointUrl": "opc.tcp://192.168.100.20:50000"}`\
```
Comment (karok2m) [inline docs/modules/publisher.md]: Let's put the request payload on a new line in a full code block (with `json`). that would make it easier to spot at a glance and it should also look more consistent with the other methods as their payloads are usually big enough to put them in a code block.

```
{
  "EndpointUrl": "opc.tcp://192.168.100.20:50000"
}
```

File: docs/modules/publisher.md (hunk 2 above)
Comment (dacolgit) [reply docs/modules/publisher.md:305]: added json

File: docs/modules/publisher.md (hunk 3 above)
Comment (dacolgit) [reply docs/modules/publisher.md]: done

**Review by karok2m — APPROVED** [review]

**Pull request merged**